### PR TITLE
Pack command support TOSCA based Shells as well

### PR DESCRIPTION
**Pull request by borismod**
## Description
Pack command determines whether it's a TOSCA based shell or not and pack it accordingly.

## Related Stories
connect #54 

## Breaking
NO

---
This change is [Reviewable](https://reviewable.io/reviews/qualisystems/shellfoundry/67)

### CODE REVIEW

File: shellfoundry/commands/pack_command.py [hunk 1]
```diff
@@ -12,18 +14,27 @@ def __init__(self):
...
         self.package_builder = PackageBuilder()
         self.dependencies_packager = PythonDependenciesPackager()
+        self.shell_package_builder = ShellPackageBuilder()
 
     def pack(self):
+
+        current_path = os.getcwd()
+
+        if self.shell_package_builder.is_tosca_based_shell(current_path):
+            self.shell_package_builder.pack()
```
Comment (doppleware) [inline shellfoundry/commands/pack_command.py]: how come you're not passing current_path to package builder?

Comment (borismod) [inline shellfoundry/commands/pack_command.py]: @doppleware we never send the current path to the command in order to keep all the logic inside the commands/utilities and not in bootstrap.py

**Comment by doppleware** [comment]
Review status: 0 of 10 files reviewed at latest revision, 2 unresolved discussions.

---

*[shellfoundry/utilities/shell_package_builder.py, line 25 \[r4\]](https://reviewable.io:443/reviews/qualisystems/shellfoundry/67#-KQQ39fbMxDA4A6hd1YJ:-KQQ39fbMxDA4A6hd1YK:bep2357) ([raw file](https://github.com/qualisystems/shellfoundry/blob/4d872d6cd3b7816294577c3795b801020224ac04/shellfoundry/utilities/shell_package_builder.py#L25)):*
> ```Python
>         :return:
>         """
>         package_path = 'shell-package'
> ```

use os.tempdir, you can use the TempDirContext module I created inside the ShellFoundry utilities folder

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/qualisystems/shellfoundry/67)*

**Comment by borismod** [comment]
Review status: 0 of 10 files reviewed at latest revision, 2 unresolved discussions.

---

*[shellfoundry/utilities/shell_package_builder.py, line 25 \[r4\]](https://reviewable.io:443/reviews/qualisystems/shellfoundry/67#-KQQ39fbMxDA4A6hd1YJ:-KQQ7ttF4B1Ft7oSCm8c:b-896fix) ([raw file](https://github.com/qualisystems/shellfoundry/blob/4d872d6cd3b7816294577c3795b801020224ac04/shellfoundry/utilities/shell_package_builder.py#L25)):*
<details><summary><i>Previously, doppleware (Roni Dover) wrote…</i></summary>
> use os.tempdir, you can use the TempDirContext module I created inside the ShellFoundry utilities folder
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/qualisystems/shellfoundry/67)*

**Comment by borismod** [comment]
Review status: 0 of 10 files reviewed at latest revision, 2 unresolved discussions.

---

*[shellfoundry/commands/pack_command.py, line 24 \[r1\]](https://reviewable.io:443/reviews/qualisystems/shellfoundry/67#-KQLt3VjXtEJhQC38IWG-r1-24:-KQQ82uc2cr-0s2aGEle:b-896fix) ([raw file](https://github.com/qualisystems/shellfoundry/blob/d52d4afb3679752cbf4521378339425d0531e102/shellfoundry/commands/pack_command.py#L24)):*
<details><summary><i>Previously, borismod (Boris Modylevsky) wrote…</i></summary>
> @doppleware we never send the current path to the command in order to keep all the logic inside the commands/utilities and not in bootstrap.py
</details>
Done.

---

*[shellfoundry/utilities/shell_package_builder.py, line 25 \[r4\]](https://reviewable.io:443/reviews/qualisystems/shellfoundry/67#-KQQ39fbMxDA4A6hd1YJ:-KQQ86VOQgCFZInWzfha:b-896fix) ([raw file](https://github.com/qualisystems/shellfoundry/blob/4d872d6cd3b7816294577c3795b801020224ac04/shellfoundry/utilities/shell_package_builder.py#L25)):*
<details><summary><i>Previously, borismod (Boris Modylevsky) wrote…</i></summary>
> Done.
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/qualisystems/shellfoundry/67)*

**Pull request merged**